hazelcast/src/test/java/com/hazelcast/client/starter/HazelcastClientStarterTest.java

```diff
@@ -54,26 +51,6 @@ public void testClientLifecycle() {
         }
     }
 
-    @Test
-    public void testClientMap() {
```

Comment:
what was the reason to delete this test?

Reply:
@ihsandemir the whole `HazelcastStarter` framework in the [original migration tool design](https://github.com/hazelcast/hazelcast-enterprise/tree/master/docs/design/migration-tool#testing-criteria) has been repurposed for executing WAN compatibility tests across major versions. One of the consequences was that several related "normal" compatibility tests stopped working one way or another. I guess this is one of those tests.